src/aat/resources/features/F-107/F-107.feature

```diff
@@ -0,0 +1,53 @@
+@F-107
+Feature: F-107: Organisation Policies on Cases
+
+    Background: Load test data for the scenario
+    Given an appropriate test context as detailed in the test data source
+
+    @S-925
+    Scenario: must successfully create a case with a case type containing some OrganisationPolicy fields
+      Given a user with [an active profile in CCD]
+      And a successful call [to create a token for case creation] as in [F-107_Case_Data_Create_Token_Creation]
+      When a request is prepared with appropriate values
+      And the request [contains some OrganisationPolicy fields with all correct values]
+      And it is submitted to call the [Submit Case Creation as Caseworker] operation of [CCD Data Store]
+      Then a positive response is received
+      And the response has all other details as expected
+
+    @S-927
+    Scenario: must successfully update OrganisationPolicy fields on a case
+      Given a user with [an active profile in CCD]
+      Given a successful call [to create a token for case creation] as in [F-107_Case_Data_Create_Token_Creation]
```

Comment:
Token creation for subsequent case creation can be skipped in the scenarios, by following a naming convention for the token creation data ids, or (as of ver 3.6.0) by specifying a prerequisite in the test data file of depending call.

Reply:
Fixed. Removed the token creation step. Used the naming convention for token creation.